### PR TITLE
Average velocity

### PR DESCRIPTION
**Pull request by Voldoned**

### CODE REVIEW

File: symplyphysics/laws/thermodynamics/average_square_of_velocity.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+                           validate_input, validate_output)
+
+# Description
+## For an ideal gas, the average square of velocity is directly proportional to its temperature and inversely proportional to the molar mass of the gas: <V^2> = 3 k T / m
+## Where:
+## <V^2> - average square of velocity
+## T - temperature in gas
+## m - mass of a gas
```
Comment (blackyblack) [inline symplyphysics/laws/thermodynamics/average_square_of_velocity.py:10]: Is it mass of gas or molar mass?

File: test/thermodynamics/average_square_of_velocity_test.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    errors,
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.laws.thermodynamics import average_square_of_velocity as velocity_law
+
+# Description
+## The ideal gas has a temperature of 300 K. The mass of gas is equal to 32 gram. Then the average square of velocity should be 3.881 * 1E-21 (m/s)^2
```
Comment (blackyblack) [inline test/thermodynamics/average_square_of_velocity_test.py:13]: Where did you find this example? I'm not sure numbers are valid.

File: test/thermodynamics/average_square_of_velocity_test.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    t = Quantity(300 * units.kelvin)
+    Args = namedtuple("Args", ["m", "t"])
+    return Args(m=m, t=t)
+
+
+def test_basic_average_square_velocity(test_args):
+    result = velocity_law.calculate_average_square_velocity(test_args.t,
+                                                            test_args.m)
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.velocity ** 2)
+    result_pressure = convert_to(result, (units.meter / units.second) ** 2).evalf(5)
```
Comment (blackyblack) [inline test/thermodynamics/average_square_of_velocity_test.py:28]: Why pressure?

File: symplyphysics/laws/thermodynamics/average_square_of_velocity.py (hunk 1 above)
Comment (Voldoned) [reply symplyphysics/laws/thermodynamics/average_square_of_velocity.py:10]: The mass of the gas. I was wrong last time.

Comment (blackyblack) [reply symplyphysics/laws/thermodynamics/average_square_of_velocity.py:10]: I'm not sure it is correct.

Comment (Voldoned) [reply symplyphysics/laws/thermodynamics/average_square_of_velocity.py:10]: I'm wrong again. I'll fix it. The mass for one atom. This is derived from the molar mass and Avogadro constant.

Comment (blackyblack) [reply symplyphysics/laws/thermodynamics/average_square_of_velocity.py:10]: Mass of molecule, I think.

Comment (Voldoned) [reply symplyphysics/laws/thermodynamics/average_square_of_velocity.py:10]: Yes: $m = \mu / N_A$.

File: test/thermodynamics/average_square_of_velocity_test.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    units,
+    Quantity,
+    SI,
+    convert_to,
+)
+from symplyphysics.laws.thermodynamics import average_square_of_velocity as velocity_law
+
+# Description
+## Example from https://easyfizika.ru/zadachi/molekulyarnaya-fizika/opredelit-srednyuyu-kvadratichnuyu-skorost-molekul-vodoroda/
+## The ideal gas has a temperature of 273 K. The mass of one atom is equal to 3,32 * 10^(-27) kg. Then the average square of velocity should be 3.403 * 10^6 (m/s)^2
```
Comment (blackyblack) [inline test/thermodynamics/average_square_of_velocity_test.py:14]: It is not mass of an atom of hydrogen.

Comment (Voldoned) [reply test/thermodynamics/average_square_of_velocity_test.py:14]: If molar mass of molecule $H_2$? 
$\mu(H_2) = 2 * \mu(H) = 2 * 0.001 = 0.002 (\text{kg/mol})$
Then
$m = \mu(H_2) / N_A = 0.002 / (6.022 * 10^{23}) = 3.32 * 10^{-27} (\text{kg})$

Comment (blackyblack) [reply test/thermodynamics/average_square_of_velocity_test.py:14]: Yes. It's mass of molecule. Molecule has 2 atoms of hydrogen.

Comment (Voldoned) [reply test/thermodynamics/average_square_of_velocity_test.py]: Thank you for your patience and understanding!

**Review by blackyblack — APPROVED** [review]

**Pull request merged**